include/platform/radio.h

```diff
@@ -164,7 +189,7 @@ ThreadError otPlatRadioSetShortAddress(uint16_t aShortAddress);
 /**
  * Enable the radio.
  *
- * @retval ::kThreadError_None  Successfully transitioned to Idle.
+ * @retval ::kThreadError_None  Successfully transitioned to Sleep.
```

Comment:
Nope, we transition to Sleep. Please take a look at the FSM description in radio.h